src/components/tooltip/tooltip.test.tsx

```diff
@@ -16,49 +11,43 @@ const mockedGuid = "guid-12345";
 jest.mock("../../__internal__/utils/helpers/guid");
 (guid as jest.MockedFunction<typeof guid>).mockImplementation(() => mockedGuid);
 
-const positions: TooltipPositions[] = ["top", "bottom", "left", "right"];
-
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-function render(props: Partial<TooltipProps> = {}) {
-  const children = <div data-role="tooltip-target">foo</div>;
-  const message = "foo";
-
-  return renderRTL(
-    <Tooltip {...props} message={message || props.message}>
-      {props.children || children}
+function renderTooltip(props: Partial<TooltipProps> = {}) {
```

Comment:
comment (non-blocking): same comment about abstracting the render, happy to sort all this later though 👍

Reply:
Yeah, this file needs a good review due to the nested structure and styling rules, so I'll leave this for now.